javascript/chronometer.js

```diff
@@ -1,26 +1,39 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = 0;
   }
+
   startClick(callback) {
-    // ... your code goes here
+    this.intervalId = setInterval(() => {
+    this.currentTime += 1
+  }, 1000);
+  }
+
+    getMinutes() {
+      let min = Math.floor(this.currentTime / 60);
+      return min;
+    }
+
+    getSeconds() {
+      let min = Math.floor(this.currentTime / 60);
+      let sec = this.currentTime - min * 60;
+      return sec;
```

Comment:
Interesting approach to solve this, but it works! :) A simpler one would be to use the % operator to get the remainder of the currentTime.
`this.currentTime % 60`